plugin/ale/sign.vim

```diff
@@ -40,6 +40,34 @@ execute 'sign define ALEWarningSign text=' . g:ale_sign_warning
 \   . ' texthl=ALEWarningSign'
 sign define ALEDummySign
 
+function! ale#sign#GetStatuslineStr()
+	" Returns a formatted string that can be integrated in the 
+	" statusline
+
+	redir => signs
+	silent exec 'sign place buffer=' . bufnr('%')
+	redir END
+
+	let [errors, warnings] = [0, 0]
```

Comment:
I say just put each variable declaration on its own line. They don't need to be combined into one line like this.